src/OrchardCore.Modules/OrchardCore.Media/Views/Admin/Index.cshtml

```diff
@@ -2,7 +2,7 @@
 <style asp-src="~/OrchardCore.Media/Styles/media.min.css" debug-src="~/OrchardCore.Media/Styles/media.css"></style>
 
 <script at="Foot">
-    initializeMediaApplication(true, '@Url.Action("MediaApplication", "Admin", new { area = "OrchardCore.Media" })', '@Context.Request.PathBase.ToString()');
+    initializeMediaApplication('@T["Media Library"]', true, '@Url.Action("MediaApplication", "Admin", new { area = "OrchardCore.Media" })', '@Context.Request.PathBase.ToString()');
```

Comment:
This probably shouldn't be passed in directly, but read from a hidden div on the `MediaApplication.cshtml` view

Reply:
I Hope to have understand well your request 😅. I Hope to have understand well your request. I have opted to don't read the text from an hidden input but I have choose to read the localized string from the root folder link that already render it.

Thank you.